functions/openhab.bash

```diff
@@ -108,8 +108,8 @@ openhab_setup() {
   cond_redirect usermod -aG "$gid" "$username" &> /dev/null
 
   echo -n "$(timestamp) [openHABian] Setting up openHAB service... "
-  if ! zram_dependency install ${ohPkgName}; then return 1; fi
-  if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
+  if ! cond_redirect zram_dependency install ${ohPkgName}; then return 1; fi
+  if ! cond_redirect systemctl -q daemon-reload; then echo "FAILED (daemon-reload)"; return 1; fi
```

Comment:
didn't check but doesn't zram_dependency do that ?

Reply:
Do what?

Reply:
Figured it out, yes actually so we can remove that legacy code. I'll do that as a separate patch